### PR TITLE
Added edges to graph. Also added script to download mock models

### PR DESCRIPTION
**Pull request by daming-lu**
Add edges to graph. For example, a node as following:

```
{
  "attribute":[
    {
      "ints":[
        "2",
        "2"
      ],
      "name":"strides",
      "type":"INTS"
    },
    {
      "ints":[
        "0",
        "0",
        "0",
        "0"
      ],
      "name":"pads",
      "type":"INTS"
    },
    {
      "ints":[
        "3",
        "3"
      ],
      "name":"kernel_shape",
      "type":"INTS"
    }
  ],
  "input":[
    "data_0",
    "conv1_w_0",
    "conv1_b_0"
  ],
  "name":"",
  "opType":"Conv",
  "output":[
    "conv1_1"
  ]
}
```

will have 4 edges:

```
[
{'source': u'data_0', 'target': 'node_0', 'label': 'label_0'}, 
{'source': u'conv1_w_0', 'target': 'node_0', 'label': 'label_1'}, 
{'source': u'conv1_b_0', 'target': 'node_0', 'label': 'label_2'}, 
{'source': 'node_0', 'target': u'conv1_1', 'label': 'label_3'}
]
```

### CODE REVIEW

File: server/visualdl/mock/squeezenet_graph.json [hunk 1]
```diff
@@ -0,0 +1,2072 @@
+{
+    "input": [
```
Comment (Superjomn) [inline server/visualdl/mock/squeezenet_graph.json]: this JSON file is too large, can we use a smaller model or just some pseudo-data?
just checking the logic do not need a full model.

If others need a complex JSON, give them offline seems more reasonable.

File: server/visualdl/mock/inception_graph.json [hunk 2]
```diff
@@ -0,0 +1,4425 @@
+{
+    "input": [
+        {
+            "data_type": "FLOAT",
+            "name": "conv1/7x7_s2_w_0",
+            "shape": [
+                "64",
+                "3",
```
Comment (Superjomn) [inline server/visualdl/mock/inception_graph.json]: too large JSON file.

**Review by Superjomn — CHANGES_REQUESTED** [review]
Json files are too large.

File: server/visualdl/mock/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## Note
```
Comment (jacquesqiao) [inline server/visualdl/mock/README.md:1]: need to add a download script

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**